modules/ocf/facts.d/polkit-priv-drop

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
```

Comment:
Suggestion: perhaps a check for the existence of the polkit user might be simpler? (I don't have strong feelings about this fwiw)